articles/why-nodejs-good-for-online-store/index.md

```diff
@@ -0,0 +1,38 @@
+
+### Introduction
+[Node.js](https://nodejs.dev/learn) is a JavaScript environment used to develop high-performance applications that are scalable and handles multiple I/O operations. E-commerce systems involve several modules working together. Each module, with its complexities, makes Node.js perfect for developing such a system.
+### Advantages of building with Node.js
+
+**Performance** - Node.js gives better results with comparatively lower costs as it supports multitasking. It is convenient compared to other languages since an e-commerce store involves several operations such as adding items to the cart, changing the item features, choosing preferred payment options. From the performance point of view, the technology to be used has to serve such tasks efficiently, and Node.js does this effectively due to its ability to handle different operations simultaneously, thus making it a good choice for an online store.
```

Comment:
`It is convenient compared to other languages since an e-commerce store`
You compared Node.js to languages.
Node.js is not a language. Could you correct it.